.vscode/launch.json

```diff
@@ -1,126 +1,130 @@
-                "PORT": "9000",
-                "UCP_CONFIG": "${workspaceFolder}/cmd/ucpd/ucp-self-hosted-dev.yaml"
-            }
-        },
-        {
-            "name": "Launch Controller",
-            "type": "go",
-            "request": "launch",
-            "mode": "auto",
-            "preLaunchTask": "Build Radius (all)",
-            "program": "${workspaceFolder}/cmd/controller/main.go",
-            "cwd": "${workspaceFolder}",
-            "args": [
-                "--config-file",
-                "${workspaceFolder}/cmd/controller/controller-self-hosted.yaml",
-                "--cert-dir",
-                ""
-            ],
-            "env": {
-                "RADIUS_ENV": "self-hosted"
-            }
-        },
-        {
-            "name": "Launch Deployment Engine",
-            "type": "coreclr",
-            "request": "launch",
-            "preLaunchTask": "Build Deployment Engine",
-            "program": "${workspaceFolder}/../deployment-engine/src/DeploymentEngine/bin/Debug/net6.0/arm-de.dll",
-            "args": [],
-            "cwd": "${workspaceFolder}/../deployment-engine/src/DeploymentEngine",
-            "stopAtEntry": false,
-            "env": {
-                "ASPNETCORE_URLS": "http://localhost:5017",
-                "ASPNETCORE_ENVIRONMENT": "Development",
-                "KUBERNETESBICEPEXTENSIBILITYURL": "http://localhost:5017/api",
-                "RADIUSBACKENDURI": "http://localhost:9000",
-            },
-        },
-        {
-            "name": "Debug Bicep generator integration tests",
-            "type": "node",
-            "request": "launch",
-            "runtimeArgs": [
-              "--inspect-brk",
-              "${workspaceRoot}/hack/bicep-types-radius/src/autorest.bicep/node_modules/.bin/jest",
-              "--runInBand",
-              "--no-cache"
-            ],
-            "cwd": "${workspaceFolder}/hack/bicep-types-radius/src/autorest.bicep/src",
-            "console": "integratedTerminal",
-            "internalConsoleOptions": "neverOpen",
-            "sourceMaps": true
-        },
-    ],
-    "compounds": [
-        {
-            "name": "Launch Control Plane (all)",
-            "configurations": [
-                "Launch UCP",
-                "Launch Applications RP",
-                "Launch Controller",
-                "Launch Deployment Engine",
-            ],
-            "stopAll": true
-        }
-    ],
-    "inputs": [
-        {
-            "id": "cliArgs",
-            "type": "promptString",
-            "description": "Args for launching Radius cli. Use --cwd to set the working directory.",
-            "default": "init --full"
-        }
-    ]
-}
\ No newline at end of file
+  // Use IntelliSense to learn about possible attributes.
+  // Hover to view descriptions of existing attributes.
+  // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+  "version": "0.2.0",
+  "configurations": [
```

Comment:
This was an autoformat. I think @ytimocin updated this file during the addition of prettier, but it didn't get checked in. 

We intentionally `.gitignore` this file, so it requires some effort to change 😆